README.md

```diff
@@ -1,33 +1,99 @@
-# Strigo.io GitHub Managed Exercises Repository
+# Strigo.io GitHub Managed Resources Repository
 
 ## Quick Start Guide
 
 1. Fork this repository or create a new `.strigo/config.yml` file in your existing repository.
-1. Configure the `.strigo/config.yml` with your exercise settings:
+2. Customize the `.strigo/config.yml` file with the lab resources and exercise configurations that suit your training needs:
 
    ```yaml
    ---
    classes:
-     - exercises:
+     - lab_resources:
+         - display_name: "AWS EC2 Instance"
+           platform_type: "lab"
+           image_platform: "linux"
+           view_interface: "desktop"
+           cloud_provider: "aws"
+           aws_vm_definition:
+             machine_size: "t2.micro"
+             ami_region_mapping:
+               eu-west-1:
+                 image_id: "ami-0abcdef1234567890"
+                 pool_size: 10
```

Comment:
we don't support pooling for class resources.
It will have to be introduced when we implement the lab resource definition GH import

Reply:
Will remove this (and other unsupported features or feature we don't want to expose yet) in the following PR